Client/Assets/WindowCloseHelper.js

```diff
@@ -0,0 +1,8 @@
+(function () {
+  var _close = window.close;
+  window.close = function () {
+    console.log("Yo I am going to close");
```

Comment:
Not sure if you want me to give this a full review since you said it's a WIP but flagged for `review?`. I'll just treat it like a regular review in case you want to land.

Remove this line (obviously :D)